packages/ckeditor5-ui/src/editorui/editorui.ts

```diff
@@ -669,6 +671,22 @@ export default abstract class EditorUI extends /* #__PURE__ */ ObservableMixin()
 		data.viewportOffset.left += configuredViewportOffset.left;
 		data.viewportOffset.right += configuredViewportOffset.right;
 	}
+
+	private _bindBodyCollectionWithFocusTracker() {
+		const body = this.view.body;
+
+		for ( const view of body ) {
+			this.focusTracker.add( view.element! );
+		}
+
+		body.on<CollectionAddEvent<View>>( 'add', ( evt, view ) => {
+			this.focusTracker.add( view.element! );
+		}, { priority: 'low' } );
```

Comment:
Why `low` priority is needed?

Reply:
@oleq I actually don't know why I added this priority here. I tested both with AI Assistant and in a dropdowns manual test and it works fine on normal priority. Unit tests pass too. Speaking of, I missed a unit test for this functionality.

One thing that I thought about is that items are rendered on normal priority, so maybe I wanted to be sure they are rendered already. But as I said, this happens on normal priority in `ViewCollection`. Since `BodyCollection` extends `ViewCollection` and rendering callback is added in constructor, any other callback on normal priority added for `BodyCollection#event:add` will fire after rendering.